src/zino/tasks/bfdtask.py

```diff
@@ -61,6 +61,8 @@ def _update_state(self, port: Port, new_state: BFDState):
         if port.bfd_state:
             if port.bfd_state.session_state != new_state.session_state:
                 self._create_or_update_event(port, new_state)
+        elif new_state.session_state != BFDSessState.UP:
+            self._create_or_update_event(port, new_state)
```

Comment:
It would be good to also then update the docstring of this function, because as I understand it this contradicts it

Reply:
Did you mean the comment right beneath the docstring? Yes that one shouldve been removed, done it now. Docstring itself still seems OK (although a bit vague)

Reply:
Yes, sorry, I meant the comment :D That's the consequences of reviewing a PR "just before I go home" :grin: